docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -0,0 +1,41 @@
+= Restore a PostgreSQL service
+
+IMPORTANT: The restores can only be triggered from an instance in the same namespace.
+If the old instance has already been deleted the backups are deleted as well.
+
+. TODO: get the name of the backup you want to restore...
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to restore a PostgreSQL instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: pgsql-app1-restore
+spec:
+  parameters:
+    restore:
+      claimName: pgsql-app1-prod <1>
+      backupName: pgsql-app1-prod-dx4fd-2023-02-17-08-50-01 <2>
+    backup:
+      schedule: '0 22 * * *'
+    service:
+      majorVersion: "15"
+      pgSettings:
+        timezone: Europe/Zurich
+    size:
+      cpu: "600m"
+      memory: "3500Mi"
+      disk: "80Gi"
+  writeConnectionSecretToRef:
+    name: postgres-creds-restored
+  compositionRef:
+    name: vshnpostgresrestore.vshn.appcat.vshn.io <3>
+----
+<1> The claim name of the instance you want to restore from
```

Comment:
Is there any reason why this is called "claim" other than because Crossplane names it like that? This isn't too user friendly IMHO, we should rather name this "instance".